### PR TITLE
Update runner-installation.adoc

### PR DESCRIPTION
**Pull request by thomasr8**
Added brief section on draining.

# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

**Comment by thomasr8** [comment]
This fixes #4945

File: jekyll/_cci2/runner-installation.adoc [hunk 1]
```diff
@@ -418,6 +418,17 @@ Here are few valid examples:
 
 NOTE: The default value is 5 hours.
 
+===== Customizing job timeouts and drain timeouts
+
+If you would like to customize the job timeout setting, you can "drain" the job by sending the Launch Agent a termination (TERM) signal, which then causes the Launch Agent to attempt to gracefully shutdown. When this TERM signal is received, the launch agent enters 'draining' mode, preventing the Launch Agent from accepting any new jobs, but still allowings any current active job to be completed. At the end of "draining," the Launch Agent then signals the Task Agent to cancel any active job (by sending it a TERM signal).
```
Comment (glenjamin) [inline jekyll/_cci2/runner-installation.adoc:423]:
```suggestion
If you would like to customize the job timeout setting, you can "drain" the job by sending the Launch Agent a termination (TERM) signal, which then causes the Launch Agent to attempt to gracefully shutdown. When this TERM signal is received, the launch agent enters 'draining' mode, preventing the Launch Agent from accepting any new jobs, but still allowing any current active job to be completed. At the end of "draining," the Launch Agent then signals the Task Agent to cancel any active job (by sending it a TERM signal).
```

File: jekyll/_cci2/runner-installation.adoc [hunk 2]
```diff
@@ -418,6 +418,17 @@ Here are few valid examples:
...
+
+NOTE: If the Task Agent does not exit after a brief period, the Launch Agent will manually kill it by sending it a KILL signal.
```
Comment (glenjamin) [inline jekyll/_cci2/runner-installation.adoc:425]:
```suggestion
NOTE: If the Task Agent does not exit a brief period after the TERM, the Launch Agent will manually kill it by sending it a KILL signal.
```

File: jekyll/_cci2/runner-installation.adoc [hunk 3]
```diff
@@ -418,6 +418,17 @@ Here are few valid examples:
...
+
+Draining can end in one of two ways:
+
+* The task has been in the draining state for longer than the configured `max_run_time`.
+* A TERM signal is received by the Launch Agent during "draining".
```
Comment (glenjamin) [inline jekyll/_cci2/runner-installation.adoc:430]:
```suggestion
* An additional TERM signal is received by the Launch Agent during "draining".
```

**Review by glenjamin — APPROVED** [review]

**Review by annabaker — APPROVED** [review]

**Comment by thomasr8** [comment]
This fixes #4945

**Pull request merged**